components/scream/data/scream_output.yaml

```diff
@@ -19,15 +19,15 @@ Fields:
       - bm
       - precip_liq_surf
       - precip_ice_surf
-      - qv_prev_micro_step
       - eff_radius_qc
       - eff_radius_qi
-      - micro_liq_ice_exchange
-      - micro_vap_liq_exchange
-      - micro_vap_ice_exchange
-      - tke
+      - nc_activated
+      - ni_activated
+      - nc_nuceat_tend
       - cldfrac_liq
+      - cldfrac_ice
       - cldfrac_tot
+      - tke
       - eddy_diff_mom
       - sgs_buoy_flux
       - inv_qc_relvar
```

Comment:
are we using `inv_qc_relvar` for any of our calculations now? If not, I think we can remove.

Reply:
Both of your suggestions are great, @crterai . I've just pushed a commit with both of them.